### PR TITLE
TC_03.06.012 | Create Multiconfiguration project

### PR DESCRIPTION
**Pull request by Deneshik**
https://trello.com/c/9QIqmvo9/365-tc0306012-create-multiconfiguration-project

### CODE REVIEW

**Review by AramH20 — APPROVED** [review]

File: cypress/e2e/multiconfigProject33.cy.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress"/>
+
+import multiconfigProject33 from "../fixtures/multiconfigProject33.json"
+
+describe('multiconfigProject33', () => {
+    
+    it('TC_03.06.012 | Create Multiconfiguration project', () => {
+        cy.get('a[href="/view/all/newJob"]').click();
+        cy.get('input#name').type(multiconfigProject33.projectName);
+        cy.get('li[tabindex="0"] span').contains('Multi-configuration project').click();
```
Comment (S-Lanchik) [inline cypress/e2e/multiconfigProject33.cy.js:10]: Put 'Multi-configuration project' to fixtures.

**Comment by S-Lanchik** [comment]
Edit your PR name (delete 'added').

File: cypress/e2e/multiconfigProject33.cy.js (hunk 1 above)
Comment (Deneshik) [reply cypress/e2e/multiconfigProject33.cy.js:10]: So I added "Multi-configuration project" to the fixtures "multiconfigProject33" and take the projectName from there. Or did I misunderstand you?

Comment (S-Lanchik) [reply cypress/e2e/multiconfigProject33.cy.js:10]: On line 10 you are using text 'Multi-configuration project', not from fixtures

Comment (Deneshik) [reply cypress/e2e/multiconfigProject33.cy.js]: @S-Lanchik I have corrected it, please check! Thanks!

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**